src/json_schema/types.rs

```diff
@@ -1,14 +1,18 @@
+//! Static collection of regular expressions for JSON and format types used
+//! in generating a regular expression string based on a given JSON schema.
+
 // allow `\"`, `\\`, or any character which isn't a control sequence
 pub static STRING_INNER: &str = r#"([^"\\\x00-\x1F\x7F-\x9F]|\\["\\])"#;
 pub static STRING: &str = r#""([^"\\\x00-\x1F\x7F-\x9F]|\\["\\])*""#;
-
 pub static INTEGER: &str = r#"(-)?(0|[1-9][0-9]*)"#;
 pub static NUMBER: &str = r#"((-)?(0|[1-9][0-9]*))(\.[0-9]+)?([eE][+-][0-9]+)?"#;
 pub static BOOLEAN: &str = r#"(true|false)"#;
 pub static NULL: &str = r#"null"#;
 
+/// Default whitespace pattern used for generation a regular expression from JSON schema.
```

Comment:
generation -> generating

You can add a comment for context:

"We chose to impose this default pattern because we found empirically that letting the model choose the number of white spaces and new lines led to pathological behaviors (especially for small models). See https://github.com/dottxt-ai/outlines/issues/484 for instance."

Reply:
Great, I will update 👍